### ISSUE TITLE
Clickable pins and tags

### ISSUE THREAD

**Issue by katescarth** [katescarth/literary-halifax-mobile-app#14]
Not really an issue, but wanted to confirm that under the "Landmarks" tab, the landmark pins are clickable but once you're in a particular landmark, under the "map" tab, the pins are not clickable, right? I kept trying to click the pins on the map tab, so just wanted to mention that but I think ppl will realize what's going on pretty quickly.

This is probably a xml/cvs issues: under "Anne of Green Gables Goes to Halifax," tag include "tag01" and <i> </i>. Need to get rid of those. 

Otherwise looking good! Can't wait to get some real content in there!

**Comment by David-Owen-Walker** [comment]
That's correct, markers are only clickable when the map is displaying several of them, which  is in Landmarks and in each tour. Hopefully people will catch on, but the only way to be sure is to put it in their hands. It does seem like there's not much harm in making them clickable, though. Worst case scenario, it just gets ignored.

tag01 isn't an error, as far as I know it's just an uncreative name. \<i>Anne of the Island\</i> does present an issue, though. \<i> tags are used to italicise text, but we can't italicise text inside tag selection. This probably won't be an issue, just something to be aware of.

**Comment by katescarth** [comment]
If the map tab/single landmark were clickable, it would lead back to description tab (I guess that would make the most sense). I think for now, it makes sense to leave as it is and let's see what other people think!

And good to know re: tags. I'll keep this in mind for when we do xml/cvs files again. Thanks, David!
